### ISSUE TITLE
[Feature]: Adding clearer instructions on how to add more in depth logging for running litellm proxy in the console

### ISSUE THREAD

**Issue by ChrisDryden** [BerriAI/litellm#760]
### The Feature

Currently the errors show:
```
LiteLLM.Info: If you need to debug this error, use `litellm.set_verbose=True'.
```

But theres no clear way to do this in the cli, theres also no indication of a parameter to set in the litellm --help

### Motivation, pitch

For people who are using the local server to run thing like AutoGen with models that have issues with the code it would help more with debugging if it was easier to add more debugging information

### Twitter / LinkedIn details

chrisdryden

**Comment by ghost** [comment]
hey @ChrisDryden  add the `--debug` flag to your cli command. I believe you can access this via `litellm --help`. 

e.g.: 

```
$ litellm --model bedrock/anthropic.claude-v2 --debug
```

Maybe a QOL improvement here could be - for the CLI tool, also print that.

**Comment by ChrisDryden** [comment]
Whoops yea that was what I was looking for, I'll submit a CR to improve the description of debug and change that statement in the logs to say --debug

**Comment by ghost** [comment]
@ChrisDryden  Fix pushed - https://github.com/BerriAI/litellm/commit/4e1885734a16cd3f946e698d67009d6b7b76e153 

Will update this ticket once it's in prod.

**Comment by ghost** [comment]
Also pushed fixes for bedrock exception mapping - 142750adff29dd22e49819b3d480e28a5ecaf761 and 1c4dd0671b71385fd7181f6480cb379d245e1553

including returning the correct error message if region is not set.
